SECURITY.md

```diff
@@ -1,43 +1,65 @@
-# Security Contact
+# Security
 
 **Contact:** security@novu.co
 
-At Novu, we prioritize the security of our systems. While we strive to make our systems as secure as possible, vulnerabilities can still exist. If you discover a vulnerability, we kindly request your assistance in helping us enhance our security measures and protect our clients.
+Based on [https://supabase.com/.well-known/security.txt](https://supabase.com/.well-known/security.txt)
```

Comment:
Please remove this line.

Reply:
@Cliftonz 
I have the required changes as per your request.
Please merge the PR now.